src/aat/resources/features/F-056/F-056.feature

```diff
@@ -0,0 +1,52 @@
+@F-056
+Feature: F-056: Submit event creation as Citizen
```

Comment:
Just a suggestion: Can we rename it to "Submit event creation as a Citizen". It feels more readable.

Reply:
👍 updated